onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.h

```diff
@@ -3,7 +3,7 @@
 
 #pragma once
 
-#if !defined(ORT_MINIMAL_BUILD)
+#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)
 
```

Comment:
OK for now. Longer term we may need to be more specific. You'd only want to include the selectors if you a) have a QDQ format model and b) are planning to use something other than the CPU EP to run it.